openssl-sys/src/lib.rs

```diff
@@ -417,6 +417,8 @@ extern "C" {
     #[cfg(feature = "aes_ctr")]
     pub fn EVP_aes_128_ctr() -> *const EVP_CIPHER;
     // fn EVP_aes_128_gcm() -> EVP_CIPHER;
+    #[cfg(feature = "aes_cfb")]
+    pub fn EVP_aes_128_cfb() -> *const EVP_CIPHER;
```

Comment:
If I'm reading the OpenSSL changelogs right, I don't think this needs to be behind a feature since it was added way back in version 0.9.7.